### DISCUSSION TITLE
Synctex with keyboard only

### DISCUSSION THREAD

**Discussion by mathmd** [ahrm/sioyek#1233] [Q&A] [answered]
From my understanding, the only way to use inverse search from Sioyek back to editor is through the command `synctex_under_cursor`. This can be inconvenient since, we need to place the cursor above the exact same location we want to be synced. Is there a way I can do this without using mouse?

(edited: for example, using synctex under the visual mark instead of cursor?)

**Comment by ahrm** [comment, marked as answer]
In the development branch we have a `synctex_under_ruler` command which does exactly that. Preview builds of the development branch are available here: https://github.com/ahrm/sioyek/releases/tag/sioyek3-alpha0.
